samples/java/hibernate/src/main/java/com/google/cloud/postgres/models/TracksId.java

```diff
@@ -0,0 +1,44 @@
+package com.google.cloud.postgres.models;
+
+import java.io.Serializable;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class TracksId implements Serializable {
```

Comment:
Also here: I think it would be good to add a small comment on why we are creating this embeddable ID.